osf/migrations/0042_add_quickfiles.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-06-13 17:35
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations, models
+
+from osf.models import OSFUser, QuickFiles
+from osf.models.quickfiles import get_quickfiles_project_title
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def add_quickfiles(*args, **kwargs):
+    users_without_quickfiles = OSFUser.objects.exclude(created__type='osf.quickfiles')
+    logger.info('About to add QuickFiles for {} users.'.format(users_without_quickfiles.count()))
+
+    total_quickfiles_to_create = users_without_quickfiles.count()
+    quickfiles_to_create = []
+    for user in users_without_quickfiles:
+        quickfiles_to_create.append(
```

Comment:
There are > 70K users on prod, so this list will take up quite a bit of memory. It would be a good idea to paginate. The page size can be pretty large (> 1000).